staging/traefik-forward-auth/templates/hooks.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         helm.sh/chart: {{ include "traefik-forward-auth.chart" . }}
     spec:
       restartPolicy: Never
+      serviceAccountName: {{ include "traefik-forward-auth.fullname" . }}
```

Comment:
this isnt created a hook creation time. So this wont work. Do we still need the clusterrole to create the secrets?